org-cyf-itp/content/onboarding/success/index.md

```diff
@@ -0,0 +1,32 @@
++++
```

Comment:
It feels like there is a category of objectives missing here around "knowing how CYF works and how to interact with the curriculum"? Could we add some things along the lines of: Understand the rhythm of a week, have done a pair programming session, have managed a backlog, can explain why we do this whole backlog/planning board thing?

Reply:
Yeah I agree, but the success page needs to be simpler not more complicated, so can you pick ONE thing from this list?

Reply:
Take a look  - I've pulled out the how our curriculum works and next steps workshops from the old curriculum

Reply:
I'm not sure I see this?